bin/start-chronos.bash

```diff
@@ -1,27 +1,39 @@
 #!/bin/bash
-set -o errexit -o nounset -o pipefail
+set -o errexit -o pipefail
 
-chronos_home="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd -P )"
-echo "Chronos home set to $chronos_home"
-export JAVA_LIBRARY_PATH="/usr/local/lib:/lib:/usr/lib"
-export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-/lib}"
-export LD_LIBRARY_PATH="$JAVA_LIBRARY_PATH:$LD_LIBRARY_PATH"
```

Comment:
So this statement ```if [ -z "$1" ] || [ "${1:0:1}" == '-' ]``` will not work if args 0 and 1 don't exist. So ``` flags=( "$@" )``` should be fine if we get into the if statement without adding nounset. I think we should add ```set -o errexit -o nounset -o pipefail``` after the if statement you mentioned. The only thing it will be catching is ```exec "$@"``` at the end (assuming it didn't go into the if).

This is a nit, but I would like to be as careful as possible here.